commit bbb1d61457a3eec74bd420f43818fcc2803f495d
Author: Benjamin Chrétien <chretien@lirmm.fr>
Date:   Fri Jan 17 23:20:11 2014 +0100

    Fix Boost error.
    
    Error was: ‘lock_guard’ is not a member of ‘boost’

diff --git a/src/wg_eeprom.cpp b/src/wg_eeprom.cpp
index 28e058d..cbd3404 100644
--- a/src/wg_eeprom.cpp
+++ b/src/wg_eeprom.cpp
@@ -35,6 +35,8 @@
 #include "ethercat_hardware/wg_eeprom.h"
 #include "ros/ros.h"
 
+#include <boost/thread.hpp>
+
 namespace ethercat_hardware
 {
 
